### PR TITLE
Pcos Detection

### PR DESCRIPTION
**Pull request by payalsinha12**
# Pull Request for ML-Crate 💡

## Issue Title: [PROJECT ADDITION]- PCOS DETECTION 

- **Info about the related issue (Pcos Detection )** : helps to detect that the person is suffering from Pcos or not.
- **Name:** Payal SInha
- **GitHub ID:** https://github.com/payalsinha12
- **Idenitfy yourself:** A Web Developer and Graphic Designer

Closes: #624 

### Describe the add-ons or changes you've made 📃

I made a project on Pcos Detection using the dataset taken from Kaggle.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Follow the project structure and update your files accordingly.
@payalsinha12

**Comment by abhisheks008** [comment]
https://github.com/abhisheks008/ML-Crate/tree/main/ASELSAN%20Stock%20Prices

Follow the project structure and put your files.

**Review by payalsinha12 — COMMENTED** [review]
changes done.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Remove the `Pcos Detection.ipynb` file, which is outside the project folder.
2. File names should be as `README.md`, `requirements.txt`.
3. Update the README file as per the given template, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md. Make it eye pleasing as it's shown in the template, using different emojis.
4. Remove the `.csv` file from the Model folder. as you have already shared the dataset file inside the Dataset folder.
5. Upload the EDA results inside the Images folder including the confusion matrices, accuracy graphs.
6. Add different visualization images inside the README file and showcase different aspects of the dataset.

@payalsinha12

**Review by payalsinha12 — COMMENTED** [review]
changes done

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Remove the images from the Model folder and put them inside the Images folder.
2. Remove the `desktop.ini` file.

**Comment by payalsinha12** [comment]
you told me to add different visualization images inside the README file. the images present inside the model folder is actually present in the README file now you are asking to delete the pictures what should i do ?? and i removed the desktop.ini file @abhisheks008

**Review by payalsinha12 — COMMENTED** [review]
done

**Comment by abhisheks008** [comment]
> you told me to add different visualization images inside the README file. the images present inside the model folder is actually present in the README file now you are asking to delete the pictures what should i do ?? and i removed the desktop.ini file @abhisheks008

If you remove the images from the Model folder and put them inside the Images folder, you can still hyperlink the images and use them inside the README irrespective of the folder.

**Review by payalsinha12 — COMMENTED** [review]
Hyperlink added @abhisheks008

**Comment by payalsinha12** [comment]
@abhisheks008 hey please review this

**Comment by abhisheks008** [comment]
image

You haven't fixed this.

**Comment by abhisheks008** [comment]
> image
> You haven't fixed this.

Create a table for showing the accuracy scores, it's not feasible to show the accuracy scores like this. It's a minor markdown change you need to do.

**Comment by payalsinha12** [comment]
please tell in which folder i have to create the table suggest the proper path for it @abhisheks008

File: Pcos Detection/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* Seaborn
+* Numpy
+* Sklearn
+
+📊**Exploratory Data Analysis Results**
+<a href="C:\Users\KIIT\Desktop\SSOC'24\ML-Crate\Pcos Detection\Images\Screenshot 2024-06-07 114101.png">Confusion Matrix</a>
+<a href="C:\Users\KIIT\Desktop\SSOC'24\ML-Crate\Pcos Detection\Images\Screenshot 2024-06-07 114118.png">Class Dostribution using SMOTE</a>
+
+
+📈 **Performance of the Models based on the Accuracy Scores**
```
Comment (abhisheks008) [inline Pcos Detection/Model/README.md:48]: The markdown script you have written is not creating a table, instead it is showing one after another.

This is what you need to fix.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Review by payalsinha12 — COMMENTED** [review]
Check now @abhisheks008

**Comment by payalsinha12** [comment]
@abhisheks008 please review it and merge it

**Comment by abhisheks008** [comment]
image

Images are not visible.

**Review by payalsinha12 — COMMENTED** [review]
all the changes are done successfully . I've uploaded the drive link and it is working now @abhisheks008

**Comment by payalsinha12** [comment]
please review it asap @abhisheks008

**Comment by abhisheks008** [comment]
No need to add drive link. You can just hyperlink the images from the Images folder to the README.

**Comment by payalsinha12** [comment]
but what is the need of changing again if that link is working ultimately we want to see the pictures and it is visible then why i have to change it . please assign me as those links are also working now

**Review by payalsinha12 — COMMENTED** [review]
i have done the changes now please review it and merge it @abhisheks008

**Review by abhisheks008 — APPROVED** [review]
Approved @payalsinha12

**Comment by abhisheks008** [comment]
Hi @payalsinha12 sorry for the inconvenience caused from my end. Please share your email address for further communications. 

I have added your points in the excel sheet, no worries now.

**Comment by payalsinha12** [comment]
> Hi @payalsinha12 sorry for the inconvenience caused from my end. Please share your email address for further communications.
> 
> I have added your points in the excel sheet, no worries now.

sinhapayal2913@gmail.com 
Thank you for the help

**Pull request merged**